docs/devGuide/bootcamp/contributeToDocs.md

```diff
@@ -91,6 +91,40 @@ Check that the following are true:
 
 </box>
 
+## Adding Intra-Site Links to Documentation
+
+Due to the way MarkBind's documentation is deployed, links from the developer guide that refer to the user guide, or vice versa, have to be differentiated with tags. MarkBind's documentation currently has 3 tags:
```

Comment:
Can we add a note here that intra-site links within UG and DG need not add the tags